vmc/resource_vmc_cluster.go

```diff
@@ -24,7 +24,15 @@ func resourceCluster() *schema.Resource {
 		Update: resourceClusterUpdate,
 		Read:   resourceClusterRead,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ",")
```

Comment:
what does idParts mean? can I see an example in main.tf?

Reply:
can acceptance tests be added for import?

Reply:
> what does idParts mean? can I see an example in main.tf?

It is not a part of configuration so will not be defined in main.tf file. The terraform import command is used to support the use case wherein a user loses the terraform.tfstate file and already has a resource like an SDDC creatied. The only thing we need to do in order to use the terraform import command is to define an empty block in the main.tf file for the resource. This is something the command supports and if the empty block is not defined then you see an error like this : 
resource address "vmc_cluster.cluster_1" does not exist in the configuration.

Before importing this resource, please create its configuration in the root module. For example:

resource "vmc_cluster" "cluster_1" {
  # (resource arguments)
}

Reply:
> can acceptance tests be added for import?

I checked the other providers and mainly see the CRUD operations defined in the acceptance tests.  I don't know if there is a way to add an acceptance test using a terraform import command. I will get back to you on this